__tests__/api.test.js

```diff
@@ -0,0 +1,51 @@
+const request = require("supertest");
+const app = require("../routes/api.router.js");
+const connection = require("../db/connection.js");
+const testData = require("../db/data/test-data/index.js");
+const seed = require("../db/seeds/seed.js");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  connection.end();
+});
+
+describe("/api/categories - GET request", () => {
+  it("responds with a status code of 200", () => {
+    return request(app).get("/api/categories").expect(200);
+  });
+
+  it("responds with an array of the category objects, each with a slug property and a description property", () => {
+    return request(app)
+      .get("/api/categories")
+      .then((response) => {
+        const { categories } = response.body;
+        expect(categories.length).toBe(4);
+        categories.forEach((category) => {
+          expect(typeof category.slug).toBe("string");
+          expect(typeof category.description).toBe("string");
+        });
+      });
+  });
+});
+
+describe("Error handling", () => {
+  it("invalid path - responds with a status code of 404 if passed a route that does not exist", () => {
+    return request(app)
+      .get("/api/invalidEndpoint")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toEqual("Invalid input");
+      });
+  });
+
+  it("invalid path - responds with a specified error message if passed a route that does not exist", () => {
+    return request(app)
+      .get("/api/invalidEndpoint")
+      .then((response) => {
+        expect(response.body.msg).toEqual("Invalid input");
+      });
+  });
```

Comment:
Nice work here Lauren 🚀 however, I think you're tests could be a little neater by condensing 4 into 2 🤔 . After you expect your error status you can continue with your .then block for your expect statements.